fbw-a380x/src/systems/instruments/src/FCU/Components/NdData.tsx

```diff
@@ -42,27 +60,131 @@ export class NdData extends DisplayComponent<NdDataProps> {
+            width="620px"
+            class={{ hidden: this.lightsTest }}
+            src="/Images/fbw-a380x/fcu/ARPT.png"
+          />
+          {/* LIGHT TESTS */}
+          <img
+            style="position: absolute; top: 0; left: 0"
+            width="620px"
+            class={{ hidden: this.lightsTest.map(SubscribableMapFunctions.not()) }}
+            src="/Images/fbw-a380x/fcu/TEST.png"
+          />
+          <img
+            style="position: absolute; top: 0; left: 890px"
+            width="620px"
+            class={{ hidden: this.lightsTest.map(SubscribableMapFunctions.not()) }}
+            src="/Images/fbw-a380x/fcu/TEST.png"
+          />
+          <img
+            style="position: absolute; top: 0; left: 1840px"
+            width="620px"
+            class={{ hidden: this.lightsTest.map(SubscribableMapFunctions.not()) }}
+            src="/Images/fbw-a380x/fcu/TEST.png"
+          />
+          <img
+            style="position: absolute; top: 0; left: 2790px"
+            width="620px"
+            class={{ hidden: this.lightsTest.map(SubscribableMapFunctions.not()) }}
+            src="/Images/fbw-a380x/fcu/TEST.png"
+          />
+          <img
+            style="position: absolute; top: 0; left: 3740px"
+            width="620px"
+            class={{ hidden: this.lightsTest.map(SubscribableMapFunctions.not()) }}
+            src="/Images/fbw-a380x/fcu/TEST.png"
+          />
         </div>
 
         <div class="BottomRow">
           <img
             style="position: absolute; top: 0; left: 0"
             width="620px"
+            class={{ hidden: this.lightsTest }}
             src={this.navaidMode1.map((v) => NdData.NAVAID_1_IMAGES[v])}
           />
-          <img style="position: absolute; top: 0; left: 890px" width="620px" src="/Images/fbw-a380x/fcu/WX.png" />
-          <img style="position: absolute; top: 0; left: 1840px" width="620px" src="/Images/fbw-a380x/fcu/TERR.png" />
-          <img style="position: absolute; top: 0; left: 2790px" width="620px" src="/Images/fbw-a380x/fcu/TRAF.png" />
+          <img
+            style="position: absolute; top: 0; left: 890px"
+            width="620px"
+            class={{ hidden: this.lightsTest }}
+            src="/Images/fbw-a380x/fcu/WX.png"
+          />
+          <img
+            style="position: absolute; top: 0; left: 1840px"
+            width="620px"
+            class={{ hidden: this.lightsTest }}
+            src="/Images/fbw-a380x/fcu/TERR.png"
+          />
+          <img
+            style="position: absolute; top: 0; left: 2790px"
+            width="620px"
+            class={{ hidden: this.lightsTest }}
+            src="/Images/fbw-a380x/fcu/TRAF.png"
+          />
           <img
             style="position: absolute; top: 0; left: 3740px"
             width="620px"
+            class={{ hidden: this.lightsTest }}
             src={this.navaidMode2.map((v) => NdData.NAVAID_2_IMAGES[v])}
```

Comment:
Yeah, like this. For the navaids specifically you can just add an extra state for light test.